.github/workflows/cmake-multi-platform.yml

```diff
@@ -40,6 +40,16 @@ jobs:
             c_compiler: clang
             cpp_compiler: clang++
             bundled_protobuf: 'OFF'
+          - os: ubuntu-20.04
+            build_type: Release
+            c_compiler: clang-12
```

Comment:
what is default compiler there?

Reply:
@vitalybuka Clang 11 from focal-updates. Ubuntu 20.04 natively has clang-10 and clang-12 from "focal" and clang-11 from "focal-updates".